test/e2e/kubelet.go

```diff
@@ -228,4 +404,87 @@ var _ = framework.KubeDescribe("kubelet", func() {
 			})
 		}
 	})
+
+	// Delete nfs server pod after another pods accesses the mounted nfs volume.
+	framework.KubeDescribe("host cleanup with volume mounts [HostCleanup]", func() {
```

Comment:
Since this code specifically tests a particular volume plugin, how hard it is to move this into its own file? such `kubelet_nfs.go` or something?